tools/ucd2c.pl

```diff
@@ -1,12 +1,15 @@
 use v5.14;
 use warnings; use strict;
 use Data::Dumper;
-use Carp qw(cluck);
+use Carp qw(cluck croak);
 $Data::Dumper::Maxdepth = 1;
 # Make C versions of the Unicode tables.
 
-# Before running, download zip files from http://www.unicode.org/Public/zipped/
-# and extract them in UNIDATA
+# Before running, downloading UNIDATA.zip from http://www.unicode.org/Public/zipped/
+# and extract them into UNIDATA in the root of this repository.
+
+# Download allkeys.txt from http://www.unicode.org/Public/UCA/latest/
+# and place into a folder named UCA under the UNIDATA directory.
```

Comment:
Not important for this PR, but wonder if we should stick a shell script under tools to wget/curl and unzip them...